src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static List<User> users;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        users = new ArrayList<>();
+        users.add(new User("UserX2023", "pass6789", 35));
+        users.add(new User("JohnDoe23", "john4567", 28));
+        users.add(new User("Anna1998", "password123", 25));
+    }
+
+    @Test
+    void userNotInDb_isOK() {
+        User actual = registrationService.register(new User("Ivan2004", "1234567", 20));
+        assertEquals(new User("Ivan2004", "1234567", 20), actual);
```

Comment:
The method name `userNotInDb_isOK` should reflect the state and the expected behavior more precisely. Consider renaming it to something like `register_newUser_userRegistered`.